src/containers/Network/UpgradeStatus.tsx

```diff
@@ -44,11 +48,11 @@ export const UpgradeStatus = () => {
 
     return tempData
       .map((item) => ({
-        label: item.server_version ? item.server_version : ' N/A ',
+        label: item.server_version ? item.server_version.trim() : ' N/A ',
         value: (item.count * 100) / total,
         count: item.count,
       }))
-      .sort((a, b) => (a.label > b.label ? 1 : -1))
+      .sort((a, b) => (isLaterVersion(a.label, b.label) ? 1 : -1))
```

Comment:
Will there be a big performance hit as a result of this? Is there any way we can mitigate that, maybe with some extra backend processing?

Reply:
I dont think it would affect the performance by much since isLaterVersion has constant runtime.

Reply:
@ckniffen please correct me if I'm wrong

Reply:
@pdp2121 best way is to put a breakpoint and see how often it is hit

Reply:
I fixed it so it ran only when refetching happens (every 5 seconds).

Reply:
It might have constant runtime, but it still has to do more processing than just comparing `label`, and it has to do it for each element in the list. 
Does this run nlogn times due to the sort, or is it just n times?

Reply:
Yes, this runs nlogn times due to the sort

Reply:
@ckniffen any ideas?